### ISSUE TITLE
Blender freezes completely with ROCM 3.7 

### ISSUE THREAD

**Issue by xcom169** [RadeonOpenCompute/ROCm#1210]
Blender freezes completely my whole PC with ROCM 3.7 on Manjaro Linux.

My test file: 
https://cloud.blender.org/p/gallery/5dd6d7044441651fa3decb56

My GPU: 
Vega 56

blender: /mnt/8942b2b9/tmp/pamac-build/hsa-rocr/src/ROCR-Runtime-rocm-3.7.0/src/core/runtime/runtime.cpp:1214: static bool rocr::core::Runtime::VMFaultHandler(hsa_signal_value_t, void*): Assertion `false && "GPU memory access fault."' failed.
zsh: abort (core dumped)  blender --debug

**Comment by ROCmSupport** [comment]
Hi @xcom169 
Thanks for reaching out.
We are able to reproduce this issue with ROCm 3.10.
I will share more details soon.

**Comment by ROCmSupport** [comment]
Hi @xcom169 
I came to know that developer is working on the fix and will update once fix is ready.
Please stay tuned
Thank you.

**Comment by ROCmSupport** [comment]
Hi All,

As per the latest information and clarity provided in our Documentation that ROCm does not support GUI applications officially.


Docs also updated accordingly @ [https://github.com/RadeonOpenCompute/ROCm#hardware-and-software-support](url)

Hardware and Software Support
ROCm is focused on using AMD GPUs to accelerate computational tasks such as machine learning, engineering workloads, and scientific computing. In order to focus our development efforts on these domains of interest, ROCm supports a targeted set of hardware configurations which are detailed further in this section.
**Note: The AMD ROCm™ open software platform is a compute stack for headless system deployments. GUI-based software applications are currently not supported.**

**Comment by ROCmSupport** [comment]
We are going to rephrase the text about GUI apps in our rocm documentation.
We have come up with some plans to handle GUI apps in a way.
But I am marking this as duplicate of #1106 and hence not reopening for now.
All progress can be seen at #1106 
Thank you.
